sssd_test_framework/utils/authentication.py

```diff
@@ -152,6 +152,15 @@ def test_example(client: Client, ldap: LDAP, kdc: KDC):
         """
         return KerberosAuthenticationUtils(self.host, ssh)
 
+    def passwd(self) -> PasswdUtils:
```

Comment:
This does not have to be a method, you can convert it to property (like su and ssh is).

Also, add an example test to the docstring.